### DISCUSSION TITLE
Welcome to pytest-bandit Discussions!

### DISCUSSION THREAD

**Discussion by mattwwarren** [Wanderu/pytest-bandit#20] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with folks who hope to support this plugin. We hope that you:
  * Ask questions about the plugin.
  * Share ideas for improving its stability and pytest compatibility
  * Welcome others and stay open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself, how you're using the plugin, and if you're interested in helping with some of the open issues.

**Comment by tomdottom** [comment]
:wave: @mattwwarren 

I just wanted to check that this project was still considered _active_ before putting any time in to contributing to it.

Is was thinking of addressing #18 as this issue recent forced me to disable the plugin.

T

**Reply by mattwwarren** [reply]
Hi Tom,

Thank you for checking and I do hope to encourage activity to make sure people are excited to contribute. Having some help with #18 would be great. I'm happy to go over the code in more detail, if that helps your efforts. There are a few oddities (18 included) that surely haven't aged well.

Thanks,
-Matt
